lib/nexthop.c

```diff
@@ -65,6 +66,37 @@ static int _nexthop_labels_cmp(const struct nexthop *nh1,
 	return memcmp(nhl1->label, nhl2->label, nhl1->num_labels);
 }
 
+static int _nexthop_seg6local_cmp(const struct nexthop *nh1,
```

Comment:
Why underscore?

Reply:
Because existing related function `_nexthop_labels_cmp` has underscore. I'm just respecting that naming.